micrometer-api/src/main/java/io/micrometer/api/instrument/observation/ObservationRegistry.java

```diff
@@ -31,60 +31,12 @@
 
     void setCurrentObservation(@Nullable Observation current);
 
-    /**
-     * !!!!!!!!!!!!!!!!!!!! THIS IS NOT STARTED !!!!!!!!!!!!!!!!!!!!
-     * !!!!!!!!!!!!!!!!!!!! REMEMBER TO CALL START() OTHERWISE YOU WILL FILE ISSUES THAT STUFF IS NOT WORKING !!!!!!!!!!!!!!!!!!!!
-     * @param name observation name
-     * @return this
-     */
-    default Observation observation(String name) {
-        if (config().isObservationEnabled(name, null)) {
-            return new SimpleObservation(name, this, new Observation.Context());
-        }
-        return NoopObservation.INSTANCE;
-    }
-
-    /**
-     * !!!!!!!!!!!!!!!!!!!! THIS IS NOT STARTED !!!!!!!!!!!!!!!!!!!!
-     * !!!!!!!!!!!!!!!!!!!! REMEMBER TO CALL START() OTHERWISE YOU WILL FILE ISSUES THAT STUFF IS NOT WORKING !!!!!!!!!!!!!!!!!!!!
-     * @param name observation name
-     * @param context context
-     * @return this
-     */
-    default Observation observation(String name, Observation.Context context) {
```

Comment:
Good question. I was mirroring the API after `Sample` usage since `Observation` like `Sample` is not a `Meter`. While there is `registry.timer(...)` and `Timer.start(...)`, the former returns (and registers) a `Timer` while the latter doesn't register anything and returns a `Sample`. So I thought it would be better to have only the latter. My hope was also that it makes it more clear that it's not registering or giving you a `Meter` because it's not a method on `MeterRegistry`. What do you think?